src/types.rs

```diff
@@ -78,20 +140,20 @@ pub struct Response {
     pub status_code: u16,
     pub response_type: String,
     pub headers: HashMap<String, String>,
-    pub body: String,
+    pub body: ActixBytesWrapper,
     pub file_path: Option<String>,
 }
 
 #[pymethods]
 impl Response {
     #[new]
-    pub fn new(status_code: u16, headers: HashMap<String, String>, body: String) -> Self {
+    pub fn new(status_code: u16, headers: HashMap<String, String>, body: &PyBytes) -> Self {
```

Comment:
The signature of this method also has to change a little:
```suggestion
    pub fn new(status_code: u16, headers: HashMap<String, String>, body: &PyAny) -> PyResult<Self> {
```